docs/pages/application-access/guides/usage.mdx

```diff
@@ -0,0 +1,273 @@
+
+Start the service:
+
+```sh
+$ teleport start --config=/path/to/teleport.yaml
+```
+
+### Generate Token
+
+A join token is required to authorize a Teleport Application Proxy service to
+join the cluster. Generate a short-lived join token and save it for example
+in `/tmp/token`:
+
+```sh
+$ tctl tokens add \
+    --type=app \
+    --app-name=grafana \
+    --app-uri=http://localhost:3000
+```
+
+### TLS Requirements
+
+TLS is required to secure Teleport's Unified Access Plane and any connected
+applications. When setting up Teleport, the minimum requirement is a certificate
+for the proxy and a wildcard certificate for its sub-domain. This is where
+everyone will log into Teleport.
+
+In our example:
+
+* `teleport.example.com` will host the Access Plane.
+* `*.teleport.example.com` will host all of the applications e.g. `grafana.teleport.example.com`.
+
+Teleport supports accessing these applications on other domains if required.
+DNS entries must be configured and the correct certificates need to be obtained
+for this to work.
+
+```yaml
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  tunnel_listen_addr: 0.0.0.0:3024
+  public_addr: teleport.example.com:443
+  # Example of two HTTPs keypairs.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/teleport.example.com/privkey.pem
+    cert_file: /etc/letsencrypt/live/teleport.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+    cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+```
+
+<Admonition
+  type="tip"
+  title="Using certbot to obtain wildcard certs"
+>
+  Let's Encrypt provides free wildcard certificates. If using [certbot](https://certbot.eff.org/)
+  with DNS challenge, the below script will make setup easy. Update [EMAIL] and [DOMAIN]
+
+  ```sh
+  certbot certonly --manual \
+    --preferred-challenges=dns \
+    --email [EMAIL] \
+    --agree-tos \
+    --manual-public-ip-logging-ok \
+    -d "teleport.example.com, *.teleport.example.com"
+  ```
+</Admonition>
+
+### Create a User
+
+In order to be able to access the application, a Teleport user needs to have
+a role that allows them to see it.
+
+Teleport comes with a built-in `access` role that provides access to all
+applications:
+
+```sh
+$ tctl --config=/path/to/teleport.yaml users add --roles=access appuser
+```
+
+## Start Application Service with CLI Flags
+
+A Teleport Application Proxy agent can be started with a single CLI command:
+
+```sh
+$ teleport start \
+    --roles=app \
+    --token=/tmp/token \
+    --auth-server=teleport.example.com:3080 \
+    --app-name="grafana" \
+    --app-uri="http://localhost:3000" \
+    --labels=env=dev
+```
+
+Note that the `--auth-server` flag must point to the Teleport cluster's proxy
+endpoint because the application service always connects back to the cluster over
+a reverse tunnel.
+
+### Application Name
+
+When picking an application name, it's important to make sure the name will
```

Comment:
I think if you use "Who can do what" it will be a bit easier to read. For example, users can access an application. Instead of an application can be accessed.